From 994ba64549079d4b5da16bc4c2733d9c371d53eb Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Fri, 01 Nov 2013 11:38:28 +0000
Subject: coverity: init members

---
diff --git a/src/lib/FHParser.cpp b/src/lib/FHParser.cpp
index 6bd1014..bcd5625 100644
--- a/src/lib/FHParser.cpp
+++ b/src/lib/FHParser.cpp
@@ -49,7 +49,8 @@ const char *getTokenName(int tokenId)
 } // anonymous namespace
 
 libfreehand::FHParser::FHParser()
-  : m_version(-1), m_dictionary(), m_records(), m_currentRecord(0),
+  : m_input(0), m_collector(0),
+    m_version(-1), m_dictionary(), m_records(), m_currentRecord(0),
     m_offsets(), m_fhTailOffset(0), m_pageInfo()
 {
 }
--
cgit v0.9.0.2-2-gbebe
